ts/packages/ts/src/stores/KeyValue.ts

```diff
@@ -99,7 +99,7 @@ export abstract class KeyValueConfigStore extends ConfigStore {
         return;
       }
 
-      await this.upsert(key, JSON.stringify(value));
+      await this.upsert(key, _.isPlainObject(value) ? JSON.stringify(value) : String(value));
```

Comment:
The following is better in my opinion:
```suggestion
      await this.upsert(key, typeof value === 'string' ? value : JSON.stringify(value));
```

This will ensure any non-string value will always get JSON.stringified instead of having `String()` convert something that `JSON.parse` might not be able to handle in some edge cases.

Reply:
I agree. Still, Ran specifically asked me to use _.isPlainObject() so please decide between you two and i'll implement here and in line 90